docs/custom-flows/error-handling.mdx

```diff
@@ -97,4 +97,31 @@ const signIn = await client.signIn.create({
   .then((result) => console.log("result", result))
   .catch((err) => console.error("error", err.errors[0].message));
 ```
-</CodeBlockTabs>
\ No newline at end of file
+</CodeBlockTabs>
+
+## Special error cases
+
+### User locked
+
+If you have Account Lockout enabled on your instance and the user reaches the maximum allowed attempts for password or
+backup code verification, you will receive an HTTP status of 403 (Forbidden) and the following error payload:
+
+
+```json
+{
+    "errors": [
+        {
+            "message": "Account locked",
+            "long_message": "Your account is locked. You will be able to try again in 30 minutes. For more information, please contact support.",
+            "code": "user_locked",
+            "meta": {
+                "duration_in_seconds": 1800
+            }
+        }
+    ]
+}
+```
+
+Note: Actual remaining duration will vary depending on your configuration.
```

Comment:
Let's add a link to the final configuration docs location.

Reply:
Where is the canonical location?

Reply:
Ah, you mean the customization section